app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,41 @@
+class PostsController < ApplicationController
+  before_action :logged_in_user, only: [:new, :create]
+
+  def new
+    @post = Post.new
+     
+  end
+
+  def create
+    @post = Post.new(post_params)
+    # p "check-params-post-create"
```

Comment:
Please remove commented out code:
 It’s better to delete commented code. If you need to review a previous version of your code, use git history